lib/Models/GlobeOrMap.js

```diff
@@ -91,6 +91,16 @@ GlobeOrMap.prototype._createEntityFromImageryLayerFeature = function(feature) {
         }
     };
 
+    // TODO: replace the above with this:
+    //
+    // entity.description = feature.description;  // already defined by the new Entity
+
+    // if (entity.propertyNames.indexOf('properties') === -1) {  // not defined yet, but could be in future
+    //     entity.addProperty('properties');
```

Comment:
I think we should be doing this `entity.addProperty` instead of the manual `entity.properties = {` above, and then change the code elsewhere to assume that entity.properties is never time-varying.  I think we talked about this before, but I can't remember: was there a reason that wouldn't work?